libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionsUserIdentity.java

```diff
@@ -1,19 +1,31 @@
 package uk.gov.di.ipv.core.library.auditing.extension;
 
-import lombok.Getter;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
 
+import java.util.List;
+
 @ExcludeFromGeneratedCoverageReport
-@Getter
-public class AuditExtensionsUserIdentity implements AuditExtensions {
-    private final String levelOfConfidence;
-    private final boolean ciFail;
-    private final boolean hasMitigations;
+public record AuditExtensionsUserIdentity(
+        @JsonProperty String levelOfConfidence,
+        @JsonProperty boolean ciFail,
+        @JsonProperty boolean hasMitigations,
+        @JsonInclude(JsonInclude.Include.NON_NULL) @JsonProperty(EXIT_CODE_NAME)
```

Comment:
Is this because we're using `exit_code` rather than `exitCode`?

Is there a general standard for this - given that it's a new property, we could also just call it `exitCode` for consistency!

Reply:
Mentioned in ticket as exit_code, if we r the first one to add this field we can call it as 'exitCode' not sure what is expected by txMA.

Reply:
Updated/aded the new extension field as 'exitCode' in TiCF excelsheet.